.gitignore

```diff
@@ -0,0 +1 @@
+Makefile
```

Comment:
Can this be the RTC one so it'll help with all platforms? https://devtopia.esri.com/3rdparty/zlib/blob/runtimecore/.gitignore